spring-integration-core/src/main/java/org/springframework/integration/channel/AbstractExecutorChannel.java

```diff
@@ -59,13 +59,19 @@
 public abstract class AbstractExecutorChannel extends AbstractSubscribableChannel
 		implements ExecutorChannelInterceptorAware {
 
-	protected Executor executor; // NOSONAR
+	protected @Nullable Executor executor;
 
-	protected AbstractDispatcher dispatcher; // NOSONAR
+	/**
+	 * {@code @SuppressWarnings} was used the dispatcher is initialized in each of the implementations in the {@link AbstractExecutorChannel}.   And each implementation
```

Comment:
I prefer to follow rule: One sentence per line.
However, not more that 120 symbols per line is still there.
I'm not sure, though, why `was used` since it is still here and will be here in the future.
Just present time is OK.
Or we just can remove this comment at all.
I think we may come back to this `@SuppressWarnings` when we done with nullability for all the implementations of this class.